hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/util/RestoreTool.java

```diff
@@ -487,12 +489,21 @@ private void checkAndCreateTable(Connection conn, Path tableBackupPath, TableNam
       if (createNew) {
         LOG.info("Creating target table '" + targetTableName + "'");
         byte[][] keys;
-        if (regionDirList == null || regionDirList.size() == 0) {
-          admin.createTable(htd);
-        } else {
-          keys = generateBoundaryKeys(regionDirList);
-          // create table using table descriptor and region boundaries
-          admin.createTable(htd, keys);
+        try{
+          if (regionDirList == null || regionDirList.size() == 0) {
+            admin.createTable(htd);
+          } else {
+            keys = generateBoundaryKeys(regionDirList);
+            // create table using table descriptor and region boundaries
+            admin.createTable(htd, keys);
+          }
+        } catch (NamespaceNotFoundException e){
+          LOG.warn(e.getMessage());
```

Comment:
nit: Printing the whole exception may confuse operators that something went wrong, maybe worth just log a warn saying there was no namespace and the same will be created?